service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MessageJudgeController.java

```diff
@@ -62,22 +64,15 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody Callback
 
         caseDetailsMap.put("nextHearingLabel", messageJudgeService.getNextHearingLabel(caseData));
 
-        return respond(caseDetailsMap);
-    }
-
-    @PostMapping("/validate-email/mid-event")
-    public AboutToStartOrSubmitCallbackResponse handleValidationMidEvent(@RequestBody CallbackRequest callbackRequest) {
-        CaseDetails caseDetails = callbackRequest.getCaseDetails();
-        CaseData caseData = getCaseData(caseDetails);
-        CaseDetailsMap caseDetailsMap = caseDetailsMap(caseDetails);
-
-        if (!caseData.getMessageJudgeEventData().isReplyingToAMessage()) {
-            String email = caseData.getMessageJudgeEventData().getJudicialMessageMetaData().getRecipient();
-            Optional<String> error = validateEmailService.validate(email);
+        JudicialMessageMetaData judgeMetaData = caseData.getMessageJudgeEventData().getJudicialMessageMetaData();
+        Optional<String> error = Optional.empty();
```

Comment:
if (nonNull(judgeMetaData)) {
            Optional<String> emailError = validateEmailService.validate(judgeMetaData.getRecipient());

            if(emailError.isPresent()) {
                return respond(caseDetailsMap, List.of(emailError.get()));
            }
        }